app/boot.js

```diff
@@ -5,6 +5,7 @@ var config = require("../config/config"),
 
 module.exports = {
   es: new elasticsearch.Client({
+    apiVersion: "1.7",
```

Comment:
My understanding is that the API is defined by the ES server version, so "1.7" here means v1.7 of the server. Version 8.2.0 of the NPM module defaults to the 1.7 API, which is why I pinned it to that before upgrading. The latest client now defaults to targeting the 2.1 API.

More generally, I think we need to straighten out which ES version we're using. I see you mentioned v2.1 when updating the README, should we update to that with the new server, and bump the API the client is using for consistency?